cfme/tests/services/test_service_manual_approval.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+
+from cfme.common.provider import cleanup_vm
+from cfme.infrastructure.provider import InfraProvider
+from cfme.services.catalogs.service_catalogs import ServiceCatalogs
+from cfme.automate.explorer.domain import DomainCollection
+from cfme.services import requests
+from cfme import test_requirements
+from utils.log import logger
+from utils.update import update
+from utils import testgen
+
+
+pytestmark = [
+    pytest.mark.meta(server_roles="+automate"),
+    pytest.mark.usefixtures('vm_name', 'catalog_item', 'uses_infra_providers'),
+    test_requirements.service,
+    pytest.mark.long_running
+]
+
+
+pytest_generate_tests = testgen.generate([InfraProvider], required_fields=[
+    ['provisioning', 'template'],
+    ['provisioning', 'host'],
+    ['provisioning', 'datastore']
+], scope="module")
+
+
+@pytest.fixture(scope="function")
+def domain(request):
+    dc = DomainCollection()
+    domain = dc.create(name=fauxfactory.gen_alphanumeric(), enabled=True)
+    request.addfinalizer(domain.delete_if_exists)
+    instance = dc.instantiate(name='ManageIQ')\
+        .namespaces.instantiate(name='Service')\
+        .namespaces.instantiate(name='Provisioning')\
+        .namespaces.instantiate(name='StateMachines')\
+        .classes.instantiate(name='ServiceProvisionRequestApproval')\
+        .instances.instantiate(name='Default')
+    instance.copy_to(domain)
+    return domain
+
+
+@pytest.fixture(scope="function")
+def modify_instance(domain):
+    dc = domain.parent
+    instance = dc.instantiate(name=domain.name)\
+        .namespaces.instantiate(name='Service')\
+        .namespaces.instantiate(name='Provisioning')\
+        .namespaces.instantiate(name='StateMachines')\
+        .classes.instantiate(name='ServiceProvisionRequestApproval')\
+        .instances.instantiate(name='Default')
```

Comment:
I would implement all this stuff in `copy_to()`method. So when you make `instance.copy_to(domain)`, it would return you a new instance.